R/class-plotdata-scatter.R

```diff
@@ -81,7 +83,8 @@ newScatterPD <- function(.dt = data.table::data.table(),
   }
   if (class(series$seriesY) != 'list') series$seriesY <- list(list(series$seriesY))
 
-  if (identical(attr$overlayVariable$dataShape,'CONTINUOUS')) {
+
+  if (identical(attr$overlayVariable$dataShape,'CONTINUOUS') && useGradientColorscale) {
```

Comment:
This seems like maybe it could just test for useGradientColorscale since that condition already tested for continuous vars?

Reply:
Yes! And brilliant this highlighted i had missed accounting for the rare case when there is one continuous overlay supplied as a collection variable. Many thanks to you and to our unit tests for catching this!